autoload/ref.vim

```diff
@@ -524,9 +530,18 @@ endfunction
 function! s:get_query(mode, source)
   let [source, query] = [a:source, '']
   if a:mode ==# 'normal'
-    let pos = getpos('.')
-    let res = s:sources[source].get_keyword()
-    call setpos('.', pos)
+    try
+      let pos = getpos('.')
+      let res = s:sources[source].get_keyword()
+    catch
+      let mes = v:exception
+      if mes =~# '^Vim'
```

Comment:
We should fix the default implementation of `get_keyword()` to throw a `ref:` exception, and should not wrap Vim's exception with `ref:`.

Reply:
I did so referencing this part of the current code:

https://github.com/thinca/vim-ref/blob/7c91a10cdd74883e7695759464b96c75d2b1bdbd/autoload/ref.vim#L582-L590

Should we fix this as well? And I don't understand how to fix them...

Reply:
Oh, indeed...
For now, it's okay to stay this way.